### PR TITLE
[#97] Hotfix for ipv6

### PR DESCRIPTION
**Pull request by DT3264**
[#97] Hotfix for ipv6

This PR should be enough as a hotfix for #97.

### CODE REVIEW

**Comment by DT3264** [comment]
Is DROP mode when we actually allow specific sites to access the internet and drop connection to the rest?

File: base-system/usrroot/usr/lib/hsync/libhfirewall.so [hunk 1]
```diff
@@ -14,6 +14,15 @@
 #		Enya Quetzalli <equetzal@huronos.org>
 #		Abraham Omar   <aomm@huronos.org>
 
+## Function to return the value of a given variable
+# $1 = Section
+# $2 = Variable name
+get_directives_var() {
```
Comment (equetzal) [inline base-system/usrroot/usr/lib/hsync/libhfirewall.so:20]: Why do we want this here? It is already available on the other libraries.

File: base-system/usrroot/usr/lib/hsync/libhfirewall.so [hunk 2]
```diff
@@ -76,8 +85,21 @@ firewall_accept_all() {
...
-	$(iptables --list)"
+	##############
+	###  IPv4  ###
+	##############
+	$(iptables --list)
+	##############
+	###  IPv6  ###
+	##############
+	$(ip6tables --list)
+	"
```
Comment (equetzal) [inline base-system/usrroot/usr/lib/hsync/libhfirewall.so:102]: Nice, we should copy this to the case when filtering. On line 191.

**Review by equetzal — CHANGES_REQUESTED** [review]
Overall good, but let's remove the function to the the directives variable. We can use `hos-dvar` or the `get_directives_var` that it's already included on `libhapply.so`

**Pull request merged**